index.js

```diff
@@ -965,6 +965,27 @@ class Encore {
         return this;
     }
 
+    /**
+     * Configure the style-loader.
+     *
+     * https://github.com/webpack-contrib/style-loader#Options
+     *
+     * ```
+     * Encore.configureStyleLoader(function(config) {
+     *     // change the config
+     *     // config.injectType = 'styleTag';
+     * });
+     * ```
+     *
+     * @param {function} callback
+     * @returns {Encore}
+     */
+    configureStyleLoader(callback) {
```

Comment:
Can you move this method next to `disableCssExtraction()`? Maybe we can mention `configureStyleLoader()` in the docs for `disableCssExtraction()` - I'm trying to make it easy for people to see that these 2 functions are related.

Reply:
I've done that.